Resources/Audio/Items/attributions.yml

```diff
@@ -17,3 +17,14 @@
   license: "CC-BY-SA-3.0"
   copyright: "Time immemorial"
   source: "https://github.com/tgstation/tgstation/blob/172b533d0257fcc1f8a05406f1c9fad514c14d88/sound/items/rped.ogg"
+
+- files: ["scary_horn.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "theOperand"
+  source: "https://github.com/tgstation/tgstation/blob/9a378933d2cfcb2c6692f5c60fbce979ac4e47c5/sound/spookoween/scary_horn.ogg"
+
+- files: ["airhorn.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "panurgomatic"
```

Comment:
this is a local git name that doesnt serve much for attribution, simply put "Taken from tgstation"

Reply:
made this change